locales/ar-arb/messages.po

```diff
@@ -176,7 +180,7 @@ msgid "Block contains the content (stimulus) of the item such as text or image.
 msgstr "يحتوي الحظر على محتوى (التحفيز) الخاص بالبند مثل النص أو الصورة. وهو مطلوب أيضًا للتفاعلات المضمنة."
 
 msgid "Block height (%)"
-msgstr "ارتفاع الكتلة (٪)"
+msgstr "ارتفاع الكتلة (%)"
```

Comment:
This one's not a placeholder's `%` char. It probably shouldn't have been replaced.

Reply:
updated